vncviewer/DesktopWindow.cxx

```diff
@@ -59,7 +59,7 @@ DesktopWindow::DesktopWindow(int w, int h, const char *name,
                              const rfb::PixelFormat& serverPF,
                              CConn* cc_)
   : Fl_Window(w, h), cc(cc_), firstUpdate(true),
-    delayedFullscreen(false), delayedDesktopSize(false)
+    delayedFullscreen(false), delayedDesktopSize(false), grab_keys(false)
```

Comment:
Follow the naming conventions please. This should be `grabActive` or something like that and the functions should be `toggleGrabKeyboard()` and `isGrabActive()`.